frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -38,7 +37,7 @@ const StyledCard = styled(Card)`
     flex: 1 0 0;
     padding: 8px 16px;
     flex-direction: row;
-    background: ${tokens.colors.ui.background__light.hex};
+    background: ${tokens.colors.ui.background__default.hex};
```

Comment:
When changing the background I would also change the background color of this button:
<img width="183" alt="image" src="https://github.com/user-attachments/assets/6dee9396-79d1-40e3-8c86-d3574e6c2ec7">

So it matches this:
<img width="136" alt="image" src="https://github.com/user-attachments/assets/523642a6-9be1-4d41-9d5f-ea5f630799e1">